content/common/cli-reference.md

```diff
@@ -1068,7 +1100,51 @@ TRIGGER OPTIONS:
 
 {{ blockEnd }}
 
-> **Please note:** If the `-f` or `--from` options do not accurately infer the intended registry or `.toml` file for your application, then you can explicitly specify either the `--from-registry` or  `--from-file` options to clarify this.
+> **Please note:** If the `-f` or `--from` options do not accurately infer the intended registry or `.toml` file for your application, then you can explicitly specify either the `--from-registry` or `--from-file` options to clarify this.
+
+### Watch
+
+In order to configure the files that are watched you need to configure the `component.build.watch` parameter in your spin.toml.
```

Comment:
This info might be better off on the "Building Application Code" page.

It would also be good to mention `watch` on the "Running Applications" page.

Reply:
A new issue has been created to provide additional Spin watch documentation
https://github.com/fermyon/developer/issues/550